drv/sprot-api/src/lib.rs

```diff
@@ -137,8 +141,12 @@ pub enum SprotError {
     #[idol(server_death)]
     ServerRestarted,
 
-    /// Unknown Errors are mapped to 0xff
-    Unknown = 0xff,
+    // Used if no explicit error code is available.
+    Unknown,
```

Comment:
Why remove the explicit value for `Unknown`?

In general, I'd suggest removing the `Unknown` variant entirely and using an `Option<SprotError>` instead (where `Unknown` becomes `None`).